### PR TITLE
div. Features

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: grails-app/controllers/de/laser/SurveyController.groovy [hunk 1]
```diff
@@ -2110,6 +2109,34 @@ class SurveyController {
...
+
+                    /*result.previousIesListPriceSum = 0
+                   if(result.previousSubscription){
+                       result.previousIesListPriceSum = PriceItem.executeQuery('select sum(p.listPrice) from PriceItem p join p.issueEntitlement ie ' +
+                               'where p.listPrice is not null and ie.subscription = :sub and ie.acceptStatus = :acceptStat and ie.status = :ieStatus',
+                       [sub: result.previousSubscription, acceptStat: RDStore.IE_ACCEPT_STATUS_FIXED, ieStatus: RDStore.TIPP_STATUS_CURRENT])[0] ?: 0
+
+                   }*/
+
+                    result.iesListPriceSum = PriceItem.executeQuery('select sum(p.listPrice) from PriceItem p join p.issueEntitlement ie ' +
```
Comment (croqueGrec09) [inline grails-app/controllers/de/laser/SurveyController.groovy:2125]: not fixed entitlements?

Comment (Moetez16) [reply grails-app/controllers/de/laser/SurveyController.groovy:2125]: YES

File: grails-app/services/de/laser/BootStrapService.groovy [hunk 2]
```diff
@@ -754,7 +754,7 @@ No Host Platform URL Content
...
             [ns: "ISBN", name_de: "ISBN", description_de: "Internationale Standardbuchnummer.", name_en: "ISIL", description_en: "International standard book number.", nsType: IdentifierNamespace.NS_SUBSCRIPTION, isUnique: false, isHidden: false],
-            [ns: "ISIL", name_de: "ISIL", description_de: null, name_en: "ISIL", description_en: null, nsType: IdentifierNamespace.NS_ORGANISATION, isUnique: false, isHidden: false],
+            [ns: "ISIL", name_de: "ISIL", description_de: "International Standard Identifier for Libraries and Related Organizations, entspricht dem Bibliothekssigel der Sigelstelle. Sind der Einrichtung mehrere ISIL zugeordnet ist Mehrfachangabe möglich", name_en: "ISIL", description_en: null, nsType: IdentifierNamespace.NS_ORGANISATION, isUnique: false, isHidden: false],
```
Comment (croqueGrec09) [inline grails-app/services/de/laser/BootStrapService.groovy:757]: typos in German text:
"International Standard Identifier for Libraries and Related Organizations, entspricht dem Bibliothekssigel der Sigelstelle. Sind der Einrichtung mehrere ISIL zugeordnet, ist eine Mehrfachangabe möglich."
then, provide English translation:
"International Standard Identifier for Libraries and Related Organizations, corresponds to the library identifier of the identifier authority. Are several ISILs assigned to the organisation, is multiple distribution possible."

File: grails-app/views/templates/properties/_private.gsp [hunk 3]
```diff
@@ -133,7 +133,7 @@
                                       class="ui properties form"
                                       data-update="${propertyWrapper}"
                                       data-done="c3po.initProperties('${createLink(controller:'ajaxJson', action:'lookup')}', '#${propertyWrapper}', ${tenant?.id})">
-                        <g:if test="${!(actionName.contains('survey') || controllerName.contains('survey'))}">
+                        <g:if test="${!(actionName?.contains('survey') || controllerName?.contains('survey'))}">
```
Comment (croqueGrec09) [inline grails-app/views/templates/properties/_private.gsp:136]: Can they really miss at that point?!

**Review by croqueGrec09 — COMMENTED** [review]
REMARKS!!!

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**